analyzer/runtime/extract.go

```diff
@@ -338,7 +338,7 @@ func ExtractRound(blockHeader nodeapi.RuntimeBlockHeader, txrs []nodeapi.Runtime
 				blockTransactionData.Success = nil
 			}
 
-			blockTransactionData.Method = tx.Call.Method
+			blockTransactionData.Method = string(tx.Call.Method)
```

Comment:
:information_source: oh it's a MethodName now

Reply:
Ah yes thank you, I was going to comment. It's a `MethodName`, which is defined as `type MethodName string`.